app/assets/stylesheets/components/_slider.scss

```diff
@@ -0,0 +1,199 @@
+$track-width: 16em;
```

Comment:
Is there a reason why we're defining the variables here and not in https://github.com/guidance-guarantee-programme/pension_guidance/blob/slider/app/assets/stylesheets/lib/_variables.scss ?